feature-libs/product-configurator/rulebased/core/model/configurator.model.ts

```diff
@@ -58,6 +58,12 @@ export namespace Configurator {
     images?: Image[];
   }
 
+  export interface Comment {
+    creationDate?: Date;
+    fromCustomer?: boolean;
+    text?: string;
+  }
```

Comment:
I created an interface ChatGPT4.ChatMessage in file model/chat-gpt-4.model. lets us this and in the UI controller we can map it to the MessageEvent interface of the chat component.